pkg/rpcclient/nep24/royalty.go

```diff
@@ -0,0 +1,182 @@
+// Package nep24 provides RPC wrappers for NEP-24 contracts, which define NFT royalties.
```

Comment:
Take a look at documentation of `nep17` and `nep11` packages. Extend documentation for `nep24` in compatible way.

> NFT

NEP-11